README.md

```diff
@@ -48,3 +58,14 @@ This is also meant to be exposed as a website similar to Hydra. However, there w
 - [ ] Packaging
   - [ ] Expose nix modules
 
+# Future Features
+
+- [ ] Mult-platform support (unlikely to happen unless someone donates hardware)
```

Comment:
Can some confidence be gained here through cross compiling even if hardware isn't available?

Reply:
that will likely introduce even more false negatives, as cross compilation is a less explored space than a different architecture (at least in the case of aarch64)